distributed/asyncio.py

```diff
@@ -127,10 +127,6 @@ def __init__(self, *args, **kwargs):
     async def __aexit__(self, type, value, traceback):
         await self.shutdown()
 
-    def __del__(self):
-        if self.status == 'running':
-            self.loop.asyncio_loop.run_until_complete(self.shutdown(fast=True))
```

Comment:
This can't work. If AioClient is used in an asyncio application, the loop is always running, so calling run_until_complete will raise an exception. It's the user's job to use "async with" or otherwise guarantee shutdown.